### ISSUE TITLE
Root Registry Scheme

### ISSUE THREAD

**Issue by adhdtech** [adhdtech/DRP#215]
**TLDR - Registry nodes with SRV records would:**
- **Bootstrap the mesh, give all nodes an initial entry vector**
- **Act as traffic directors for non-Registry nodes by re-targeting them to zone specific Registries.**

Currently all Registry nodes need to be published as DNS SRV records.  Instead, give implementers the option of specifying SRV records for a small set of root nodes which are in a special zone.  These would only communicate with other Registries and serve to bootstrap the mesh.  Non-Registry nodes would join the mesh by using DNS to find the roots, but would immediately be retargeted to Registries in their respective zones.

Behavior in current setup:
- Registry A starts and has a SRV record
- Registry B joins but does not have a SRV record
- Registry C joins

Registry C will not be aware of the existence of Registry B.  Registry A will only relay entries for nodes that specify it as a delegate.

Options to resolve:
- Update DRP_TopologyTracker.AdvertiseOutCheck to advertise all Registries to other Registries.
- Add a separate DRP function to pull a list of Registries which are connected but do not have SRV records.
